src/walletConnect/screens/RequestContent.tsx

```diff
@@ -70,7 +72,7 @@ function RequestContent({
     <View style={styles.container}>
       <TopBarIconButton icon={<QuitIcon />} style={styles.closeButton} onPress={handleDeny} />
       <ScrollView>
-        {dappImageUrl && (
+        {isDappListed && dappImageUrl && (
```

Comment:
I strongly think we should still display the dapp image even if not listed.
- it's not very inclusive of us to do that
- it would be impossible for new dapps to test this before being listed